packages/cli/src/parallel-tests/run-all-tests.types.ts

```diff
@@ -0,0 +1,5 @@
+export interface TestRunProgress {
+  failedTestCases: number;
+  passedTestCases: number;
+  runningTestCases: number;
+}
```

Comment:
Non-blocking: I am thinking this could also be similar data to what we push through with `putTestRunResults`.